src/Compute/v2/Models/Server.php

```diff
@@ -230,6 +230,59 @@ public function revertResize()
     }
 
     /**
+     * Gets a VNC console for a server.
+     *
+     * @param  string $type The type of VNC console: novnc|xvpvnc.
+     *                      Defaults to novnc.
+     *
+     * @return mixed
+     */
+    public function getVncConsole($type = Enum::CONSOLE_NOVNC)
```

Comment:
Type is an arg here so that user can pass custom type if needed.

Type can be either `novnc` or `xvpvnc`.  `novnc` seems to be the only type get documented according to [doc](http://developer.openstack.org/api-ref/compute/?expanded=get-vnc-console-os-getvncconsole-action-detail) so I put it as the default parameter.